multitool found apc qol

Maximal08 opened the pull request:
## About The Pull Request

Replaces the text description for the APC multitool search function with directional arrows similar to Heretic.
(If someone has better ideas for a sprite, I'm ready to listen.)

https://github.com/user-attachments/assets/833b4f10-eab2-4833-bdf6-d02bd5820f48


## Why It's Good For The Game

The old search function was inconvenient for players, it looks much better.
## Changelog
:cl:
qol: Replaces the text description for the APC multitool search function with directional arrows similar to Heretic.
image: Sprite for multitool arrow

/:cl:

Maximal08 committed:
multitool_found_apc_qol

Maximal08 committed:
run_linters_fix

deathrobotpunch commented:
make it so the APC is highlighted as well so you can find it among clutter

Maximal08 commented:
> make it so the APC is highlighted as well so you can find it among clutter

I'm not sure if I can isolate the APC itself because the algorithm that exists now is looking for the node to which it connects. But I will think about how to do it, but it will be a separate PR.

projectkepler-ru commented:
I like it but can you still keep a distances indicator? i wanna know if the nearest APC is like 220 Tiles away

Maximal08 commented:
> I like it but can you still keep a distances indicator? i wanna know if the nearest APC is like 220 Tiles away

the color indicates the range of distance to APC, I think maybe make this range more sensitive, but I don't think to return the text, because there is no particular point in it.

projectkepler-ru commented:
fuck

Maximal08 committed:
out_of_heretic

Maximal08 commented on code/modules/antagonists/heretic/heretic_living_heart.dm:
```diff
@@ -283,3 +283,9 @@
 	pixel_x = -32
 	pixel_y = -32
+
+/atom/movable/screen/multitool_arrow
```
As long as I do this, it should be fine.

Maximal08 committed:
Merge branch 'master' into pr/90896

Maximal08 committed:
fixing_merge_conflicts_and_updating_arrow_color_indication

Maximal08 commented:
> All good, just make sure to fix the merge conflicts.

Everything is ready, I fixed the conflicts and updated something for the color of the arrow to make it more intuitive.
